pkg/operation/botanist/secrets.go

```diff
@@ -88,6 +88,10 @@ func (b *Botanist) generateWantedSecrets(basicAuthAPIServer *secrets.BasicAuth,
 		kubeControllerManagerCertDNSNames = dnsNamesForService("kube-controller-manager", b.Shoot.SeedNamespace)
 		kubeSchedulerCertDNSNames         = dnsNamesForService("kube-scheduler", b.Shoot.SeedNamespace)
 
+		konnectivityServerDNSNames = append([]string{
+			common.GetAPIServerDomain(b.Shoot.InternalClusterDomain),
+		}, dnsNamesForService(common.KonnectivityServerCertName, b.Shoot.SeedNamespace)...)
```

Comment:
Hmm, shouldn't this be rather:
```suggestion
		}, dnsNamesForService("kube-apiserver", b.Shoot.SeedNamespace)...)
```
as there is no dedicated `konnectivity-server` service? Or am I missing something?

Reply:
`common.GetAPIServerDomain(b.Shoot.InternalClusterDomain)`  is set already. I added just in case we separat the APIserver from the konnectivityserver in the future.

Reply:
Hmm, `common.GetAPIServerDomain(b.Shoot.InternalClusterDomain)` does not return `kube-apiserver`, `kube-apiserver.svc` and so on...

Reply:
you are accessing the cluster over the internet, the apiserver presents the internal domain in the handshake so that does it. You would need it if you are calling kube-apiserver by it's service  name from the shoot by we don't do that right?

Reply:
Ah, now I got your intentions. I was a bit confused. Sorry :)

Reply:
no need to apologize and thanks for the feedback :)